torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -205,6 +213,41 @@ void InplaceMKLDNNSubgraph(std::shared_ptr<Graph> graph) {
   }
 }
 
+Operation createUnaryOp(
+    std::function<void(at::Tensor output, at::Tensor input)> aten_op,
```

Comment:
we are never running the op in Aten. it is invariant that if we fuse the op into the mkldnn subgraph all inputs are mkldnn tensors